PendingReleaseNotes.md

```diff
@@ -6,6 +6,8 @@
 - Creation of storage pools through the custom resource definitions (CRDs) now allows users to optionally specify `deviceClass` property to enable
 distribution of the data only across the specified device class. See [Ceph Block Pool CRD](Documentation/ceph-pool-crd.md#ceph-block-pool-crd) for
 an example usage
+- Provisioning with ceph-volume will fail if metadataDevice is specfied and the metadataDevice is not a higher performing device class than the OSD
```

Comment:
Same concern before regarding the specificities of the functionality built into Rook: "Provisioning will fail if the user specifies a metadataDevice but that device is not used as a metadata device by Ceph." (or something equivalent) should be sufficient here.